channeldb/graph.go

```diff
@@ -1459,6 +1486,31 @@ func putLightningNode(nodeBucket *bolt.Bucket, aliasBucket *bolt.Bucket, node *L
 	return nodeBucket.Put(nodePub, b.Bytes())
 }
 
+func putPartialLightningNode(nodeBucket *bolt.Bucket, aliasBucket *bolt.Bucket, node *LightningNode) error {
+	var (
+		scratch [8]byte
+		b       bytes.Buffer
+	)
+
+	nodePub := node.PubKey.SerializeCompressed()
+
+	if err := aliasBucket.Put(nodePub, []byte(node.Alias)); err != nil {
```

Comment:
Also the order here is off, the alias is written much after the public key. As a result, alias bucket shouldn't yet be updated as we don't have a valid alias for a node if we've only discovered it via a `ChannelUpdate` message.